### PR TITLE
a small fix in kubectl translations

### PR DESCRIPTION
**Pull request by AdamDang**
fix a small typo in kubectl translation file:
Line 1620: kubernets->kubernetes

### CODE REVIEW

File: translations/kubectl/it_IT/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -1617,7 +1617,7 @@ msgid ""
 msgstr ""
 "\n"
 "\t\tCrea un autoscaler che automaticamente sceglie e imposta il numero di pod "
-"che vengono eseguiti in un cluster di kubernets.\n"
+"che vengono eseguiti in un cluster di kubernetes.\n"
```
Comment (neolit123) [inline translations/kubectl/it_IT/LC_MESSAGES/k8s.po]: should be uppercase `Kubernetes`.

**Review by neolit123 — CHANGES_REQUESTED** [review]
thanks for the PR. please amend with the suggested uppercase change.

here is a guide:
https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing

File: translations/kubectl/it_IT/LC_MESSAGES/k8s.po (hunk 1 above)
Comment (AdamDang) [reply translations/kubectl/it_IT/LC_MESSAGES/k8s.po]: Thanks. I've changed all the words kubernetes in uppercase ,except those in url and file path.

**Comment by neolit123** [comment]
thanks
/ok-to-test

**Pull request closed without merging**